docs/ethical-advertising.rst

```diff
@@ -4,14 +4,35 @@ Ethical Advertising
 Read the Docs is a large,
 free web service.
 There is one proven business model to support this kind of site:
-Advertising.
-We're building an advertising model that respects users while providing value to advertisers.
+**Advertising**.
+However,
+instead of using Google,
+we are building the advertising model that we want to exist.
+We only work with companies we like and respect,
+and offer products that are relevent to developers
```

Comment:
Yea, this is just the copy around our ad campaign. We should probably have more documentation around sustainability as a whole, but the target here is mainly to act as a landing page for people who hit the little (i) info button on the ads, or for us to link to explain what we're doing.